src/javascript/campaign-url-builder/index.js

```diff
@@ -16,37 +16,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import {connect, Provider} from 'react-redux';
-import {bindActionCreators} from 'redux';
```

Comment:
Again, what's the rationale for removing redux? It's used in the other demos/tools, so removing it here doesn't save anything on file size, but it does create architectural differences between them.

Reply:
I didn't remove redux, I just removed `bindActionCreators`. The `connect` call still calls it internally, I just did a very minor shift to the idiomatic version.

Basically, you can do this:

```
Component = connect(..., {
    "action1": actionCreator1,
    "action2": actionCreator2,
})(BaseComponent)
```

And it automatically wraps each action creator in `dispatch` and merges the results into the props.

Reply:
Ok cool. It's been a while since I've looked at the developments in redux.